code/modules/projectiles/guns/specialist.dm

```diff
@@ -284,6 +284,38 @@
 	scatter = 15
 	recoil = 2
 
+//The Railgun
+
+/obj/item/weapon/gun/rifle/sniper/railgun
+	name = "\improper T-4 Projectile Railgun"
+	desc = "A highly sophisticated weapon cable of launching projectiles at very high velocity. Fires HVP."
+	icon_state = "svd"
+	item_state = "svd"
+	max_shells = 1 //codex
+	caliber = "High Velocity Projectile" //codex
```

Comment:
`desc`: `capable` instead of `cable`
Make sure there are no capitalization for caliber names. `High Velocity Projectile` to `high velocity projectile`.